examples/benchmark/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+# Copyright (C) 2016, Dennis Sitelew
+cmake_minimum_required(VERSION 3.0)
+
+if(NOT CMAKE_BUILD_TYPE)
+  message(WARNING "You shouldn't benchmark a debug build")
```

Comment:
why not? also it shouldn't be checked such way for multi-config generators, see http://stackoverflow.com/a/24470998/2288008